docs/DeveloperGuide.md

```diff
@@ -379,6 +378,69 @@ is not specified, it would be represented as `"null"`, in the `appointmentbook.j
 
 <br>
 
+## **Appendix: Parsing**
+
+### Design considerations
+
+**Aspect: How to parse the commands:**
+<br>
+Context: The commands (other than the general) have the command format: `add... ENTITY_TYPE ENTITY_ARGS`
+<br>
+* **Alternative 1 (current choice):** Parse `ENTITY_TYPE` and `ENTITY_ARGS` separately.
+    * Pros:
+        * Easier to parse
+        * Easier to debug, as the parsing is separated into different portions
+    * Cons:
+        * More verbose, and less centralised.
+* **Alternative 2**: Parse them together.
+    * Pros:
+        * Everything is parsed together, centralising the parsing logic.
+    * Cons:
+        * Harder to parse and debug
+
+<br>
+
+**Aspect: Command format (with prefixes/not with prefixes):**
```

Comment:
Is it better if we phrase it to `(with or without prefixes)`?

Reply:
Makes sense!